### ISSUE TITLE
It doesn't work as cargo subcomand

### ISSUE THREAD

**Issue by ikar49** [J-M0/cargo-alias#2]
When I build it manually and run with `cargo run --bin cargo-alias` it's work correctly.
But when I install it with `cargo install` and use as `cargo alias` it's panicked:
```
PS C:\Users\ikar49\Documents\rusty\cargo-alias> cargo alias
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', C:\Users\ikar49\.cargo\registry\src\github.com-1ecc6299db9ec823\cargo-alias-0.2.0\src\bin\cargo-alias.rs:37:59
```
When I recompile cargo-alias and put it into `.cargo/bin` with patch:
```
20.    let opt = Opt::from_args();
21.    println!("{:?}", opt);
```
It's show me:
```
PS C:\Users\ikar49\Documents\rusty\cargo-alias> cargo alias
Opt { alias: Some("alias") }
```
So. when it's called as cargo subcommand you need to parse the arguments in a different way.

**Comment by ikar49** [comment]
A fragment from the [Cargo book](https://doc.rust-lang.org/cargo/reference/external-tools.html#custom-subcommands):
> When Cargo invokes a custom subcommand, the first argument to the subcommand will be the filename of the custom subcommand, as usual. The second argument will be the subcommand name itself. For example, the second argument would be `${command}` when invoking `cargo-${command}`. Any additional arguments on the command line will be forwarded unchanged.

I haven't idea how fix it because StructOpt is using now.

**Comment by J-M0** [comment]
Fixed in 77b524cff58d94dd882a528a9b0c981a2b173a9b
